packages/cms-lib/lib/config.js

```diff
@@ -388,8 +389,12 @@ const getPortalConfig = portalId => {
   return config.portals.find(portal => portal.portalId === portalId);
 };
 
-const getPortalId = nameOrId => {
+/*
+ * Returns a portalId from the config if it exists, else returns null
+ */
+const getPortalId = (command = {}) => {
```

Comment:
Are we sure that we want to bring the concept of a command to `@hubspot/cms-lib`? I think that this also is a breaking change since the signature changes.

Reply:
Reverted this after our discussion.